tests/hubapi/demohub/bin/ssh_host_key.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCz5DGnPPVBST8h8+j7nfJ2kGEy0H/rJu1s5iORjWUHMyySZ8c6yQds4eiharqEVemNT/uO8ae06GWD3MP41am8NdScC5Xs97bLbADNn3rw+NHPLD31VmAkWRnivJLDX3tGX1bcygTDs15qilIv+umZRHxihcCBfLiwri7+abdFtmdnJWP8A6Hh2zRHsVCr7QT2IsHMPy8n1bAP8Bk+5NRIHU1eqprLIURnm1cKIIQMXrCTYtZfkmuM9mB639F2/v407G2ZJNyGeqBxgdJK3uZxwFEPWQtfK0urOvzCk5SW8XvgU7rG2nwO0z9dcpV8o1QLxcpSeX4rt/R8BnmAHOe3
```

Comment:
@remoteeng00 should above two files be committed?

Reply:
sorry, this file is added by me, I forgot to add it to .gitignore
It should be ignored